pkg/logger/logger.go

```diff
@@ -24,7 +25,7 @@ var (
 		enc.AppendString(t.Format(logTmFmtWithMS))
 	}
 	customCallerEncoder = func(caller zapcore.EntryCaller, enc zapcore.PrimitiveArrayEncoder) {
-		enc.AppendString(colorYellow + caller.TrimmedPath() + colorReset)
+		enc.AppendString(fmt.Sprintf("nsx %d [nsx-op@%d caller=\"%s\"]", os.Getpid(), getGoroutineID(), caller.TrimmedPath()))
```

Comment:
Could we add a switch like 
	flag.IntVar(&UseABNFLog)  default true
to control it?